packages/@ngtools/webpack/src/angular_compiler_plugin.ts

```diff
@@ -134,9 +134,9 @@ export class AngularCompilerPlugin implements Tapable {
     if (!this._entryModule) {
       return undefined;
     }
-    const splitted = this._entryModule.split('#');
+    const splitted = this._entryModule.split(/(#[a-zA-Z]+)$/);
```

Comment:
Names can start with an underscore or letter; and then can contain an underscore, letter, or number.

Reply:
@clydin Done